### PR TITLE
chore: Rename user-agent-js-worker to trustbloc-agent-js-worker

### LINKED ISSUE #78: Rename user-agent-js-worker to trustbloc-agent-js-worker

### PR DESCRIPTION
**Pull request by fqutishat**
closes #78

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: cmd/trustbloc-agent-js-worker/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# User Agent JS Worker
+# Trustbloc Agent JS Worker
```
Comment (troyronda) [inline cmd/trustbloc-agent-js-worker/README.md]: TrustBloc

File: cmd/trustbloc-agent-js-worker/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-User Agent Framework Go via Javascript.
+Trustbloc Agent Framework Go via Javascript.
```
Comment (troyronda) [inline cmd/trustbloc-agent-js-worker/README.md]: TrustBloc

File: cmd/trustbloc-agent-js-worker/index.html [hunk 3]
```diff
@@ -11,49 +11,49 @@
...
+    <script src="dist/web/trustblocagent.js"></script>
     <script>
-        var useragent
+        var trustblocagent
 
-        async function startUserAgent() {
-            if (useragent) {
-                console.error("user agent is already initialized")
-                document.querySelector("#output").value = "useragent is already initialized"
+        async function startTrustblocAgent() {
```
Comment (troyronda) [inline cmd/trustbloc-agent-js-worker/index.html]: TrustBloc

File: cmd/trustbloc-agent-js-worker/index.html [hunk 4]
```diff
@@ -91,7 +91,7 @@
         <tr>
             <td>
                 <fieldset>
-                    <legend style="font-size: x-large;font-weight: bold">User Agent Command Controller</legend>
+                    <legend style="font-size: x-large;font-weight: bold">Trustbloc Agent Command Controller</legend>
```
Comment (troyronda) [inline cmd/trustbloc-agent-js-worker/index.html]: TrustBloc

File: cmd/trustbloc-agent-js-worker/index.html (hunk 3 above)
Comment (fqutishat) [reply cmd/trustbloc-agent-js-worker/index.html]: done

File: cmd/trustbloc-agent-js-worker/index.html (hunk 4 above)
Comment (fqutishat) [reply cmd/trustbloc-agent-js-worker/index.html]: done

File: cmd/trustbloc-agent-js-worker/README.md (hunk 2 above)
Comment (fqutishat) [reply cmd/trustbloc-agent-js-worker/README.md]: done

File: cmd/trustbloc-agent-js-worker/README.md (hunk 1 above)
Comment (fqutishat) [reply cmd/trustbloc-agent-js-worker/README.md]: done

**Pull request merged**